### ISSUE TITLE
Has someone tried efficientnet as backbone? 

### ISSUE THREAD

**Issue by Tonyboy999** [jfzhang95/pytorch-deeplab-xception#198]
Efficientnet was considered as the best CNN in 2019, and many competitors got great rank with unet + efficientnet. I tried to use efficientnet as the backbone of this project but got poor results.

**Comment by Alphonses** [comment]
Hi,why downloaded pretrained model can not unzip?
Can you share the pretrained model for me?

**Comment by gfotedar** [comment]
I'm facing similar issues. Its worse than resnet. Were you able to figure out why?

**Comment by Tonyboy999** [comment]
Mine is not worse than resnet,it achieves similar effect as drn and a bit better than resnet. Just a little bit，but takes more time training.



发自我的iPhone


------------------ Original ------------------
From: Gaurav Fotedar ***@***.***&gt;
Date: Tue,Apr 20,2021 8:46 AM
To: jfzhang95/pytorch-deeplab-xception ***@***.***&gt;
Cc: GuangzhouTony ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [jfzhang95/pytorch-deeplab-xception] Has someone tried efficientnet as backbone?  (#198)





 
I'm facing similar issues. Its worse than resnet. Were you able to figure out why?

**Comment by gfotedar** [comment]
Did you make any changes to efficientnet or its parameters ( dropout/drop_connect ) apart from removing the last layers?

**Comment by Tonyboy999** [comment]
Almost not，I only tune the model structure by changing the low level feature layer in efficientnet. I simply extracted the feature map



发自我的iPhone


------------------ Original ------------------
From: Gaurav Fotedar ***@***.***&gt;
Date: Wed,Apr 21,2021 2:35 PM
To: jfzhang95/pytorch-deeplab-xception ***@***.***&gt;
Cc: GuangzhouTony ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [jfzhang95/pytorch-deeplab-xception] Has someone tried efficientnet as backbone?  (#198)





 
Did you make any changes to efficientnet or its parameters ( dropout/drop_connect ) apart from removing the last layers?

**Comment by Tonyboy999** [comment]
I think the effect of efficientnet may differ from different datasets.



发自我的iPhone


------------------ Original ------------------
From: Gaurav Fotedar ***@***.***&gt;
Date: Wed,Apr 21,2021 2:35 PM
To: jfzhang95/pytorch-deeplab-xception ***@***.***&gt;
Cc: GuangzhouTony ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [jfzhang95/pytorch-deeplab-xception] Has someone tried efficientnet as backbone?  (#198)





 
Did you make any changes to efficientnet or its parameters ( dropout/drop_connect ) apart from removing the last layers?

**Comment by Tonyboy999** [comment]
I think the effect of efficientnet may differ from different datasets



发自我的iPhone


------------------ Original ------------------
From: Gaurav Fotedar ***@***.***&gt;
Date: Wed,Apr 21,2021 2:35 PM
To: jfzhang95/pytorch-deeplab-xception ***@***.***&gt;
Cc: GuangzhouTony ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [jfzhang95/pytorch-deeplab-xception] Has someone tried efficientnet as backbone?  (#198)





 
Did you make any changes to efficientnet or its parameters ( dropout/drop_connect ) apart from removing the last layers?
